### PR TITLE
fix: enable help for category statuses

### PR DESCRIPTION
**Pull request by goldbez**
#### Description of changes
This PR adds functionality to the `preserveHelpInformation` function to allow it to parse the argument passed to `amplify status`. It also adds `status` subcommands to the CommandsInfo object.

#### Description of how you validated changes
Unit tests still pass

#### Checklist

- [x] PR description included
- [x] `yarn test` passes
- [ ] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/dev/CONTRIBUTING.md#tests)
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] New AWS SDK calls or CloudFormation actions have been added to relevant test and service IAM policies
- [ ] [Pull request labels](https://github.com/aws-amplify/amplify-cli/blob/dev/CONTRIBUTING.md#labels) are added

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by evcodes** [comment]
Hey, would you be able to attach some screenshots with the before and after of this change? This looks good though, I just want to see how it looks like 😄

File: packages/amplify-cli-core/src/help/commands-info.ts [hunk 1]
```diff
@@ -374,17 +374,71 @@ export const commandsInfo: Array<CommandInfo> = [
...
+        subCommandDescription: 'Displays the current status of your API',
+        subCommandUsage: 'amplify api status [-acm <table-name>]',
         subCommandFlags: [
           {
-            short: 'v',
-            long: 'verbose',
-            flagDescription: 'Shows verbose details, including cloudformation differences',
+            short: 'acm',
+            long: '',
+            flagDescription: 'Displays the access control matrix',
```
Comment (evcodes) [inline packages/amplify-cli-core/src/help/commands-info.ts:390]: never knew we even had subcommands for api

**Review by evcodes — APPROVED** [review]
LGTM

**Review by danielleadams — COMMENTED** [review]
Would you be able to add a test to this?

**Comment by goldbez** [comment]
image
image

@evcodes Here are the screenshots

**Review by danielleadams — APPROVED** [review]

**Review by evcodes — APPROVED** [review]

**Pull request merged**